base/shell/cmd/cmd.c

```diff
@@ -2308,6 +2309,23 @@ Initialize(VOID)
         ConOutPuts(_T("(C) Copyright 1998-") _T(COPYRIGHT_YEAR) _T(" ReactOS Team.\n"));
     }
 
+    /* ReactOS extension: Inform if the Asian font is not found */
+    hwndConsole = GetConsoleWindow();
+    if (GetPropW(hwndConsole, L"ReactOSCJKFontFallback"))
+    {
+        if (!*ptr)
+        {
```

Comment:
The fallback doesn't seem to depend on this variable, so you can put the warning message without checking it.